src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -269,6 +269,9 @@ def remove_group(fs, vol_spec, groupname):
     except cephfs.Error as e:
         if e.args[0] == errno.ENOENT:
             raise VolumeException(-errno.ENOENT, "subvolume group '{0}' does not exist".format(groupname))
+        elif e.args[0] == errno.ENOTEMPTY:
+            raise VolumeException(-errno.ENOTEMPTY, f"can not delete subvolume group {groupname}, "
+                                  "there's one or more subvolume(s) inside it")
```

Comment:
is it possible to mention number of subvolumes like : `cannot delete subvolume group foo since it is holding X subvolumes`

Reply:
For this we will have to make a call to `listdir` or `ls` function which I don't think makes much sense here. Our main aim is to notify users with info that subvolumegroup is not empty.